virttest/qemu_monitor.py

```diff
@@ -2087,8 +2100,19 @@ def balloon(self, size):
 
         :param size: int type values.
         """
-        self.verify_supported_cmd("balloon")
-        self.send_args_cmd("balloon value=%s" % size)
+        cmd = "balloon"
+        qmp_event = "BALLOON_CHANGE"
+        self.verify_supported_cmd(cmd)
+        # Clear the event list of QMP monitors
+        self.clear_events()
+        # Send a balloon monitor command
+        self.send_args_cmd("%s value=%s" % (cmd, size))
+        # Look for BALLOON QMP events
+        time.sleep(1)
```

Comment:
better to use wait for  here,  I'm not sure is it enough to wait for 1s to get this event.